dev-libs/nmeap/nmeap-0.3.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Extensible NMEA-0183 (GPS) data parser in standard C"
+HOMEPAGE="http://nmeap.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+DEPEND="doc? ( app-doc/doxygen )"
+
+src_prepare() {
+	default
+
+	# Repsect users CFLAGS and LDFLAGS for the static lib archive
+	sed -i -e 's/CFLAGS =/CFLAGS +=/' -e 's/-g -O0 -Werror -Wall/$(LDFLAGS)/' src/Makefile || die
```

Comment:
LDFLAGS are not correct here since you aren't linking anything.

Reply:
Okay, I thought, even when a static archive, it should be set. Removed now.